gax/src/main/java/com/google/api/gax/rpc/OperationFuture.java

```diff
@@ -52,12 +52,6 @@
   String getName() throws InterruptedException, ExecutionException;
 
   /**
-   * Returns the {@code OperationT} future of the initial request which started this {@code
```

Comment:
It felt out of place to me... It was the one piece of the interface that requires the knowledge of how the operation is structured. Who needs the lower-level details that you're referring to? If we wanted to expose the underlying operation, it would have to be exposed as `OperationSnapshot`, which exposes the result and metadata as plain `Object` instead of `ResponseT` and `MetadataT`; if we want a typed result, we'd have to create another abstraction somehow.

Reply:
Yes, this implies that each operation has a "starting operation" as its initialization. I think it is ok to have it on the surface. Without that there no way to get reasonable details about failed operation start. The metadata and response make sense only if the operation was started (i.e. initialized). This method provides the "initialization" status.

Reply:
I will add this back, but returning `ApiFuture<OperationSnapshot>`.